husky_components/overlay/main.js

```diff
@@ -463,12 +463,6 @@ define([], function() {
          * Removes the component
          */
         removeComponent: function() {
-            // todo fix bug: sometimes overlay-sandbox has own sandbox or parent-sandboxes as child which
-            // couses an endless loop. The bug can be reproduced by starting the component
-            // in a clickhandler with openOnStart-option true
-            // this.sandbox.stop();
-
-            this.sandbox.stop('*');
```

Comment:
@wachterjohannes yes and no. it seems to work properly everywhere except in the demos. But the bug in the demos just about the overlay but about stoping components in general